package.json

```diff
@@ -17,9 +17,9 @@
     "lint": "eslint . && echo 'Lint complete.'",
     "lintfix": "eslint . --max-warnings 0 --fix",
     "test:e2e": "NODE_ENV=test start-test http://localhost:5001 cy:run",
-    "e2e:audit": "NODE_ENV=test start-test http://localhost:5001 cy:run:audit",
-    "cy:run": "cypress run --browser chrome --headless",
-    "cy:run:audit": "cypress run --browser chrome --headless --env runAudit=true",
+    "e2e": "NODE_ENV=test start-test http://localhost:5001 cy:run:ci",
+    "cy:run": "cypress run --browser chrome --headless --record --key 31a18a85-d0e5-4c16-a1d4-5f0eab6a7c58  --parallel --group 'all tests'",
```

Comment:
Where does this `--key` value come from?

Reply:
mmh